src/wallet/rpcimportmulti.cpp

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) 2009-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "base58.h"
+#include "chain.h"
+#include "rpc/server.h"
+#include "init.h"
+#include "main.h"
+#include "script/script.h"
+#include "script/standard.h"
+#include "sync.h"
+#include "util.h"
+#include "utiltime.h"
+#include "wallet.h"
+
+#include <fstream>
+#include <stdint.h>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/date_time/posix_time/posix_time.hpp>
+
+#include <univalue.h>
+
+#include <boost/foreach.hpp>
+
+
+//namespaces
+using namespace std;
+
+//extern functions
+extern void ImportAddress(const CBitcoinAddress& address, const string& strLabel);
+extern void ImportScript(const CScript& script, const string& strLabel, bool isRedeemScript);
+extern bool EnsureWalletIsAvailable(bool avoidException);
+extern void EnsureWalletIsUnlocked();
+
+//binding names
```

Comment:
I feel these constants are not needed. If they were used a lot then I would agree with it.